shared_model/builders/protobuf/helpers.hpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_PROTO_BUILDER_HELPER_HPP
+#define IROHA_PROTO_BUILDER_HELPER_HPP
+
+#include "amount/amount.hpp"
+#include "primitive.pb.h"
+
+namespace shared_model {
+  namespace proto {
+    /**
+     * Initialize protobuf Amout by text number representation
+     * @param proto is Amount that is being set
+     * @param name is string representation of Amount
+     */
+    static inline void addAmount(iroha::protocol::Amount *proto,
```

Comment:
May be it would be better to put this function in a class or create a one more nested namespace for it?

Reply:
What's the point?

Reply:
Point is that at the moment addAmount is visible to everybody in shared_model::proto. I think it will be better to "hide" it a bit.

Reply:
Namespace hides exactly same as a class